ipv8/test/messaging/interfaces/udp/test_endpoint.py

```diff
@@ -61,7 +61,8 @@ async def test_send_many_messages(self):
         # range must be in [1, 51), since Asyncio transports discard empty datagrams
         for ind in range(1, 51):
             self.endpoint1.send(self.ep2_address, b'a' * ind)
-        await sleep(.05)
+        while len(self.endpoint2_listener.incoming) < 50:
+            await sleep(.02)
```

Comment:
**nit:** Probably you should specify a timeout for this test function because this loop can lead to an "endless" run.